pkg/gen/dotnet-templates/typesInput.cs.mustache

```diff
@@ -0,0 +1,63 @@
+// *** WARNING: this file was generated by the Pulumi Kubernetes client generation tool. ***
+// *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using Pulumi.Serialization;
+
+{{#Groups}}
+namespace Pulumi.Kubernetes.Types.Inputs.{{Group}} {
```

Comment:
This doesn't quite feel right - but we've diverged a fair bit on where we place these types across the .NET/TypeScript and TF/Kubernetes providers - so I'm not sure what the "right" answer is here.

Today - we have:
* TS+TF: `aws.types.input.ec2.InstanceEbsBlockDevice`
* CS+TF: `Pulumi.Aws.Ec2.Inputs.InstanceEbsBlockDevicesArgs`
* TS+Kubernetes: `kubernetes.types.input.core.v1.Pod`

By triangulating those - I think you end up with: `Pulumi.Kubernetes.Core.V1.Inputs.PodArgs` instead of the current `Pulumi.Kubernetes.Types.Input.Core.V1.PodSpec`. 

Does that wound right?  Is there a reason we were inconsistent with TypeScript on where these types live in our .NET codegen for TF?

cc @pgavlin @lblackstone @mikhailshilkov

Reply:
I've landed on `Pulumi.Kubernetes.Types.Input.Core.V1.PodSpecArgs` for now.  Open to feedback.

Reply:
That looks right to me. The CS+TF example appears to be inconsistent with the existing layout (k8s.types.[input,output].[group].[version].[kind])